code/dynamic_programming/egg_dropping_puzzle/eggdrop.c

```diff
@@ -0,0 +1,56 @@
+#include<stdio.h>
```

Comment:
Don't forget to add the OpenGenus comment as stated in the [contributing guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute).